proxy/envoy/config.go

```diff
@@ -267,6 +268,21 @@ func buildOutboundHTTPRoutes(instances []*model.ServiceInstance, services []*mod
 					routes = append(routes, buildDefaultRoute(cluster))
 				}
 
+				if service.External {
+					for _ , route := range routes {
+						for _ , cluster := range route.clusters {
+							cluster.ServiceName = ""
```

Comment:
@rshriram this is for the sidecar proxy config gen.  SDS in this case would return the IP of the egress service.  Egress proxy config gen would NOT use SDS because it will have to statically define the cluster with hosts (i.e. google.com, etc)